flux-refineck/src/fixpoint.rs

```diff
@@ -388,17 +402,30 @@ impl<'a> KVarGen for Box<dyn KVarGen + 'a> {
 
 fn fixpoint_const_map(
     genv: &GlobalEnv,
-    name_gen: &IndexGen<fixpoint::Name>,
-) -> FxIndexMap<DefId, ConstInfo> {
-    genv.map()
+    const_name_gen: &IndexGen<fixpoint::ConstName>,
+) -> FxIndexMap<Key, ConstInfo> {
+    let consts = genv
+        .map()
         .consts()
         .sorted_by(|a, b| Ord::cmp(&a.sym, &b.sym))
         .map(|const_info| {
-            let name = name_gen.fresh();
-            let cinfo = ConstInfo { name, val: const_info.val };
-            (const_info.def_id, cinfo)
-        })
-        .collect()
+            let name = const_name_gen.fresh();
+            let cinfo = ConstInfo {
+                name,
+                sym: const_info.sym,
+                sort: fixpoint::Sort::Int,
+                val: Some(const_info.val),
+            };
+            (Key::Const(const_info.def_id), cinfo)
+        });
+    let uifs = genv.uifs().map(|uif_def| {
```

Comment:
We should sort `uifs` here the same way consts are being sorted. We use the resulting list of fixpoint consts as part of the hash for a `Task` which we use for caching, so we want to have a consistent ordering here.